activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/api/ProjectRestApi.java

```diff
@@ -163,6 +165,23 @@ void exportProject(
                     required = false,
                     defaultValue = "true") boolean attachment) throws IOException;
 
+    @ApiOperation(
+            tags = PROJECTS,
+            value = "Save an project as zip file with chosen name",
+            notes = "This will create and download the zip with chosen name " +
+                    "containing the project folder and all related models.<br>")
+    @GetMapping(path = "/projects/{projectId}/saveAs")
+    void saveProjectAs(
+            HttpServletResponse response,
+            @ApiParam(value = EXPORT_PROJECT_ID_PARAM_DESCR, required = true)
```

Comment:
this is not export project call, if we re-use the same literal, let's rename it, otherwise we can create a new one